pkg/loader/loader.go

```diff
@@ -16,6 +16,87 @@ limitations under the License.
 
 package loader
 
+import (
+	"fmt"
+
+	"k8s.io/kubectl/pkg/kinflate/util/fs"
+)
+
+// Loader abstracts how bytes are read for manifest, resource, patch, or other
```

Comment:
Surprisingly, the documentation mostly talks about `kinflate` specific concepts, but none of the variable/methods/class name are referring to `kinflate`.